### ISSUE TITLE
Backpack disappears on re-log

### ISSUE THREAD

**Issue by bchawke** [DayZMod/DayZ#498]
Not sure if this is something related to the patch or the server.  I was on US434 and was having a lag issue so I logged out to lobby  for a moment and when I logged back into the game my backpack was gone.  I was wearing a soldier skin at the time, but it did NOT disappear when I changed skins, just when I re-logged.

**Comment by R4Z0R49** [comment]
Yea there's an issue we are working on it

**Comment by bchawke** [comment]
Cool, thanks.  I saw the issue with the skin change but posted this as it may have been a different problem.  It happened again tonight.  The second time I logged to lobby, left for 30 minutes or so (forgot to log off and quit DayZ), and when I reconnected from the lobby my backpack was gone again.  Both times I was logged to lobby rather than quitting the game.  Not sure if that's helpful data or not but wanted to be detailed.
